### ISSUE TITLE
Theme: Ignore List Level for Blockquotes

### ISSUE THREAD

**Issue by lloyd094** [typora/typora-issues#1833]
I am working on a CSS theme to be used for law school outlines. Due to the nature of the outlines, everything tends to be built in one giant ordered list. 

**Is it possible to override the list level for blockquotes, so blockquotes would appear the same regardless?** In other words, I'd like this:

-----

A. Topic One 
…….. a. Sub Topic A 
……..…….. i. Info paragraph
……..…….. ii. Info paragraph
> Jane Doe v. John Doe: Case, Information, Law, and Findings

…….. b. Sub Topic B
...etc

-------
Basically, I'm trying to get my blockquote to appear the same (across the whole page, as if it wasn't in a list) regardless what level the list is on, so I can then continue to the next piece on the list without trying to tab all the way over.

**Comment by lloyd094** [comment]
If it helps, I have also posted this on Stack Overflow (with very little response) and tried the suggestion to `position:absolute` and `left:0`. That suggestion did not work with Typora, and instead just made it so text started overlapping the blockquote.

**Comment by abnerlee** [comment]
Didn't come up with good solutions.

Maybe would could try this :
```css
li blockquote {
 margin-left: -18px;
}

li li blockquote {
 margin-left: -36px;
}

li li li blockquote {
 margin-left: -54px;
}
```
